README.md

```diff
@@ -101,8 +103,8 @@ When adding resources the parent object key will be used to identify the new res
   - `base`: The base path for your Drupal instance. All request for this resource will use this path. This can be different from the path used when instantiating `waterwheel`.
   - `credentials`: An object containing the `username` and `password` used to authenticate with Drupal. This can be different from the credentials used when instantiating `waterwheel`.
   - `methods`: An object containing the following keys, `GET`, `POST`, `PATCH`, `DELETE`. Each key should contain the path suffix that the action can be preformed on.
-  - `entity`: The entity that this resource should reference, ie. `node`.
-  - `options`: The bundle that this resource should reference, ie. `page`.
+  - `entity`: The entity type that this resource should reference, ie. `node`.
```

Comment:
Didn't the key above change to "entityType"?  Maybe this isn't referring to the same thing.